projects/objectnav_baselines/experiments/objectnav_base.py

```diff
@@ -5,8 +5,8 @@ class ObjectNavBaseConfig(ExperimentConfig):
     """The base object navigation configuration file."""
 
     def __init__(self):
-        self.CAMERA_WIDTH = 640
-        self.CAMERA_HEIGHT = 480
+        self.CAMERA_WIDTH = 400
```

Comment:
Why wouldn't we use a lower resolution? 400x300 has the same relative dimensions as 640x480 and we reduce everything to 224x224 at the end. I think you might be misinterpreting how these values work, they don't change how much of the scene is visible, just the image resolution. Here are two images I've saved directly from THOR at the two different resolutions.

![480x640](https://user-images.githubusercontent.com/628838/93942101-9a518c00-fce4-11ea-9fe4-f339a14fe621.png)

![300x400](https://user-images.githubusercontent.com/628838/93942096-97ef3200-fce4-11ea-98af-a8a514d5062e.png)

Reply:
I know what these values mean, and you are correct that since 400x300 is the same aspect ratio and we downscale it to a 224x224 box for resnet we should get the same result. This is *mostly* true but I was told by the thor people as well as roozbeh and ani that rendering at t higher resolution makes the borders and lines less jagged which ultimately produces a better looking downscaled image. Since the performance benefits of rendering at this lower resolution were so small ani told me to just use the higher resolution for the experiments for the paper. All that being said I do think that the difference is minimal and since we are pushing towards better performance I think that it is worth rendering at the lower res. I will ask ani and roozbeh again just in case though